Makefile

```diff
@@ -149,7 +149,8 @@ $(BOTTLES_DIR):
 bottles: $(BOTTLES)
 endif
 
-deps: | check-qt-dir deps-common bottles
+non-nim-deps: check-qt-dir bottles
```

Comment:
I would ensure if `bottles` is nim or non-nim dependency. At first glance it's for ssl libraries on mac, required for nim part. If it's like that non-nim-deps would be only check-qt-dir.

Reply:
If that's the case, then I believe we don't need `non-nim-deps`, the name sounds a bit artificial.

Reply:
Right, any idea @alexjba ?

Reply:
or @igor-sirotin

Reply:
We're using it for Qt. In fact both Qt and our nim code have a dependency on openssl (or had at some point). But this solution is quite old and from what I see in the PR is meant to support qt5.14 on really old macOS versions. Maybe it's worth revisiting it and at least test if we still need this 😄 
https://github.com/status-im/status-desktop/pull/375

Reply:
Yeah but this doesn't really matter in the context of `ui-tests` as those are run on Linux only

Reply:
But what's for sure is that it's a nim dependency. We do a static link in `configs.nim`

Reply:
Alright, I can remove it and just make the SQ/SB stuff depend on `check-qt-dir`

Reply:
Done